go-chaos/cmd/restart.go

```diff
@@ -78,26 +54,7 @@ var restartGatewayCmd = &cobra.Command{
 	Short: "Restarts a Zeebe gateway",
 	Long:  `Restarts a Zeebe gateway.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		k8Client, err := internal.CreateK8Client()
-		if err != nil {
-			panic(err)
-		}
-
-		gatewayPodNames, err := k8Client.GetGatewayPodNames()
-		if err != nil {
-			panic(err)
-		}
-
-		if len(gatewayPodNames) <= 0 {
-			panic(errors.New(fmt.Sprintf("Expected to find Zeebe gateway in namespace %s, but none found.", k8Client.GetCurrentNamespace())))
-		}
-
-		gatewayPod := gatewayPodNames[0]
-		err = k8Client.RestartPod(gatewayPod)
-		if err != nil {
-			panic(err)
-		}
-
+		gatewayPod := restartGateway(nil)
```

Comment:
:question: Why use `nil` as `gracePeriod` here and 0 for the terminate commands?

Reply:
Nil means it will fallback to the k8 default, and zero will terminate it immediately.